tasks/main.yml

```diff
@@ -10,10 +10,18 @@
 - name: Create OpenTripPlanner binary directory
   file: path={{ otp_bin_dir }} mode=0775 state=directory
 
+- name: Store derived jarfile name for convenience
+  set_fact: otp_jar_name=otp-{{ otp_version }}{{ otp_jar_suffix }}.jar
+
+- name: Check if OpenTripPlanner binary exists
+  stat: path={{ otp_bin_dir }}/{{ otp_jar_name }}
```

Comment:
It looks like we are using `stat` here to check if a previously downloaded file exists already? If so, I think we can do this with just `get_url` and avoid the `stat` task.

The `dest` argument to `get_url` can be a specific file vs. a destination path. It won't re-download the file if it is already there if we supply a `checksum`.

See also: https://github.com/azavea/ansible-papertrail/blob/develop/tasks/main.yml#L9-L13